crates/bevy_ecs/src/system/commands.rs

```diff
@@ -270,14 +446,33 @@ impl<'a, 'b> EntityCommands<'a, 'b> {
         self
     }
 
-    /// Despawns only the specified entity, not including its children.
+    /// Despawns the entity.
+    ///
+    /// See [`World::despawn`] for more details.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::prelude::*;
+    /// #
+    /// # struct CharacterToRemove { entity: Entity }
+    /// #
+    /// fn remove_character_system(
+    ///     mut commands: Commands,
+    ///     character_to_remove: Res<CharacterToRemove>
+    /// )
+    /// {
+    ///     commands.entity(character_to_remove.entity).despawn();
+    /// }
+    /// # remove_character_system.system();
+    /// ```
     pub fn despawn(&mut self) {
         self.commands.add(Despawn {
             entity: self.entity,
         })
     }
 
-    /// Returns the underlying `[Commands]`.
+    /// Returns the underlying [`Commands`].
```

Comment:
This might be nice to have some context on why you might want to use this method, to avoid baffling newcomers to Rust.